Doom The Dark Ages/doomthedarkages.asl

```diff
@@ -71,15 +96,25 @@ update
     {
         currdict[watcher.Name] = watcher.Current;
     }
+
+    //Prints the camera target to the Livesplit layout if the setting is enabled
+        if(settings["Loading"]) 
+    {
+        vars.SetTextComponent("GameState:",current.gameState.ToString());
+    }
+
+            //Prints the camera target to the Livesplit layout if the setting is enabled
+        if(settings["Mission"]) 
+    {
+        vars.SetTextComponent(" ",current.mission.ToString());
+    }
```

Comment:
in general we should try and make sure that the indentation is consistent, it can make these scripts harder to read